src/components/ContextMenu/index.tsx

```diff
@@ -3,12 +3,21 @@ import * as RadixContextMenu from '@radix-ui/react-context-menu';
 import ContextMenuItem from './ContextMenuItem';
 import ContextMenuTrigger from './ContextMenuTrigger';
 import ContextMenuContent from './ContextMenuContent';
+export interface ContextMenuProps {
+  /** Event handler called when the open state of the context menu changes. */
+  onOpenChange?: (open: boolean) => void;
 
-export type ContextMenuProps = Omit<RadixContextMenu.ContextMenuProps, 'dir' | 'modal'>;
+  /** ContextMenu content, items and trigger */
+  children: React.ReactNode;
+}
 
-const ContextMenu = ({ children, onOpenChange }: ContextMenuProps): JSX.Element => {
+const ContextMenu = ({
+  children,
+  onOpenChange,
+  ...rest
+}: ContextMenuProps): JSX.Element => {
   return (
-    <RadixContextMenu.Root onOpenChange={onOpenChange} dir="ltr" modal>
+    <RadixContextMenu.Root {...rest} onOpenChange={onOpenChange} dir="ltr" modal>
```

Comment:
is there a reason to do `{...rest}` here? rest is an empty object

Reply:
was just matching the format of `Menu` when we didnt know the issue was the import